notification/source/c/src/kaa_demo.c

```diff
@@ -46,7 +47,7 @@ static const char *color_to_str(enum color code_color)
 
 #define INPUT_LENGTH 8
 
-static int read_topic_id(void)
+static int64_t read_topic_id(void)
```

Comment:
But in #481  you've said, it is `int64_t`. Also, `int64_t` is used for topic ids in C++ SDK. (correct me if I'm wrong)